1-basics.md

```diff
@@ -14,44 +14,66 @@ kernelspec:
+  - [Files default language](https://code.visualstudio.com/docs/languages/overview)
+- [Settings sync](https://code.visualstudio.com/docs/editor/settings-sync)
+- [Profiles](https://code.visualstudio.com/docs/editor/profiles)
+  - [Python Dev profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template)
+  - [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template)
+  - [Sarah's profile collection](https://dev.to/crazy4pi314/profiles-for-fun-and-profit-how-to-use-profiles-to-customize-vs-code-57hj)
+- [Extensions marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
+  - Don't see what you need? [Make your own!](https://code.visualstudio.com/api/get-started/your-first-extension)
 
-**Exercise**: Create a new virtual environment, install a package with the env create command pallet or the built-in terminal. Then create a new Jupyter notebook or script and import the package.
+> **Exercise**: Share your profile
+>
+> Either in your local or browser-based VS Code, try customizing some settings, themes, etc. either from a blank profile or one of the templates. Then export your current editor config as a gist (or locally if you prefer). If you have the URL for your profile gist, add it to the [discussion page](https://github.com/crazy4pi314/scipy-vscode-tutorial/discussions/17).
 
-- Python extension settings: formatters / linters
-  - available as extensions: pylint, flake8, black, autopep8, ...
+## Setting up for Python development
 
-**Customize your ride** - setting up our environment for Python development
+We will set up an existing Python project and install the necessary extensions. This piece is fairy quick and self-guided.
 
-- Customizing Telemetry
-- Color themes: Dark, Colorblind
-- Auto save
-- Settings sync
-- Profiles
-- Files default language
+- [Python Environments in VS Code](https://code.visualstudio.com/docs/python/environments#_creating-environments)
+  - Default intrepreter settings
+  - [Python environment manager](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python-environment-manager)
+- Common Python extensions
+  - [Python extension pack by Don Jayamanne](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python-extension-pack)
+  - [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+  - formatters / linters: pylint, flake8, black, autopep8, ...
 
+> **Exercise**: New Python project from scratch
+>
+> In the Codespace for this workshop or locally (in a Dev Container) on your machine, Use VS Code to create a new virtual environment, install new packages with the `Python: Create Environment` command from the command pallet or the built-in terminal. Check that you have the intrepreter setup correclty by opening a new Jupyter notebook or script and import the package(s).
```

Comment:
It's good to have an exploratory component, 
but let's also have some simple task people can check off to set a minimal level to which everyone should reach; something like

```python
import emoji
print(emoji.emojize("Meet your coding best friend: VS Code :red_heart:"))
```

Reply:
Yeah good idea